pex/util.py

```diff
@@ -213,3 +215,32 @@ def named_temporary_file(*args, **kwargs):
       yield fp
   finally:
     os.remove(fp.name)
+
+
+def iter_pth_paths(filename):
+  """Given a .pth file, extract and yield all inner paths. Largely lifted from site.py."""
+  dirname = os.path.dirname(filename)
+  known_paths = set()
+
+  try:
+    f = open(filename, 'rU')  # noqa
+  except IOError:
+    return
+
+  with f:
+    for line in f:
+      line = line.rstrip()
+      if not line or line.startswith('#'):
+        continue
+      elif line.startswith(('import ', 'import\t')):
+        try:
+          exec_function(line)
+          continue
+        except Exception:
+          # Defer error handling to the higher level site.py logic invoked at startup.
+          return
```

Comment:
the error handling I'm referring to here is the dumping of the traceback and other sys.stderr logging that happens in this case in site.py [here](https://github.com/kwlzn/python-sources/blob/master/Python-2.7.13/Lib/site.py#L159-L166).

I don't think we strictly need to do the `import` bits in our function - but they're basically no-ops afaict - and without testing the import it's possible we'd consume subsequent lines that site.py itself wouldn't respect because of how it breaks out of the processing loop in this case - so I left it in because it felt more correct.

open to stronger opinions on better handling of this tho.